webapp/channels/src/components/copy_button.tsx

```diff
@@ -74,16 +76,16 @@ const CopyButton: React.FC<Props> = (props: Props) => {
             <span
                 className={spanClassName}
                 onClick={copyText}
+                aria-label={intl.formatMessage({id: getId(), defaultMessage: getDefaultMessage()})}
```

Comment:
Out of the scope, but I get shivers looking at these `getId` and `getDefaultMessage` functions... The `t` function has done a lot of harm XD

Reply:
Yeah, I wanted to change that, but I managed to keep my scope creep to just accessibility improvements